OracleCoherence/README.md

```diff
@@ -1,115 +1,193 @@
+base image (see [Oracle Java images](../OracleJava/))
 
 ## How to Build
 
-For Coherence 12.2.1.4.0, a Maven project is provided to facilitate building the Docker image.  Maven is used to pull dependent libraries
-which are then bundled into the Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints within the container.
+For Coherence 14.1.1.0.0 and 12.2.1.4.0, a Maven project is provided to facilitate building
+the Docker image.  Maven is used to pull dependent libraries which are then bundled into the
+Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints 
+within the container.
 
-### To build a Coherence 12.2.1.4.0 Docker image
+### To build a Coherence 14.1.1.0.0 or 12.2.1.4.0 Docker image
+
+#### Prerequisites for building with Maven
 
-Prerequisites for building with Maven:
 * Java 8 JDK
 * Maven 3.6.1
 
-Build steps:
+#### Build steps
+
+The following steps build a Coherence 14.1.1.0.0 Docker container as an example
 
 1. Checkout the GitHub Oracle Docker Images repository
-   
-   	`$ git clone git@github.com:oracle/docker-images.git`
+
+    ```shell script
+    git clone git@github.com:oracle/docker-images.git
+    ````
    	
-2. Go to the directory containing the Dockerfile for 12.2.1.4
+1. Go to the directory containing the Dockerfile
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker`
+    ```shell script
+    cd OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.4.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.4.0_coherence_quick_Disk1_1of1.zip**
+1. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop
+   the Coherence distribution file of your choice into this folder. The build script supports
+   building an image from either the Quick Installer, `fmw_14.1.1.0.0_coherence_quick_Disk1_1of1.zip`,
+   or the Standalone Installer, `fmw_14.1.1.0.0_coherence_Disk1_1of1.zip`.
 
-    Note that the `Dockerfile` is currently configured to use the Quick Installer.
+    > **_NOTE:_** The `Dockerfile` is currently configured to use the Quick Installer.
 
-4. Go to the maven project directory
+1. Go to the maven project directory
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0`
+    ```shell script
+    cd OracleCoherence/dockerfiles/14.1.1.0.0
+    ```
     
-5. Build the Docker image with Maven
+1. Build the Docker image with Maven
+
+    To build a Docker image using Oracle 8 JDK:
 
-    `$ mvn install`
+    ```shell script
+    mvn install
+    ```
+
+    The resulting image file will be called `oracle/coherence:14.1.1.0.0`.
+
+    To build a Docker image using GraalVM (CE) (*Coherence 14.1.1.0.0 only*)
```

Comment:
Please document the requirements for building with Graal and provide an alternative method for customers who are not permitted to pull images from the Docker Hub.

Reply:
I updated the base image instructions to reference the GraalVM instructions as well.

Reply:
I would call out the exact version of Graal that works, because there are now Dockerfiles for both Graal (JDK8) and Graal (JDK11). Be explicit about what base image is expected.

Reply:
Both Java 8 and Java 11 Graal images can be used.  I updated the base image instructions.